src/main.sh

```diff
@@ -714,13 +714,18 @@ install_package() {
 	for pkg in ${pkgname[@]}; do
 		fullver=$(NOEPOCH=1 get_full_version)
 		pkgarch=$(get_pkg_arch $pkg)
-		pkglist+=("${PKGDEST}/${pkg}_${fullver}_${pkgarch}.deb")
+		tmpfile=$(mktemp "/tmp/${pkg}_${fullver}_${pkgarch}_XXXXXX.deb")
+		cp "${PKGDEST}/${pkg}_${fullver}_${pkgarch}.deb" "${tmpfile}"
+		pkglist+=("${tmpfile}")
 	done
 
 	if ! SUDO=1 makedeb_rs install "${APTARGS[@]}" "${pkglist[@]}"; then
 		warning "$(gettext "Failed to install built package(s).")"
 		return $E_INSTALL_FAILED
 	fi
+	for pkg in ${pkglist[@]}; do
+		rm "${pkg}"
+	done
```

Comment:
It would probably be good to add some error handling in case the file gets deleted for whatever reason with an `if` block and then a call to `error`.